sample-apps/django_sample/Procfile

```diff
@@ -2,4 +2,4 @@
 proxy: npx judoscale-adapter-proxy-server
 
 # Run with --noreload to avoid multiple server processes (and multiple Judoscale reporters)
-web: python manage.py runserver $PORT --noreload
+web: gunicorn django_sample.wsgi:application --preload
```

Comment:
I discovered an unexpected side effect of this change: The sample app uses the _published_ judoscale-python package instead of the _local_ code.

I tried making some logging changes locally, and I couldn't get those changes to show up in the sample app. When I reverted the Procfile change (to use `manage.py`), my logging changes showed up.

Do you know how we can pull in the local judoscale-python code rather than the published package? I need to do that so I can test these changes.

Reply:
Are you seeing changes in this PR when you run `bin/dev`?

Here's what I see.

![CleanShot 2022-08-01 at 11 05 25@2x](https://user-images.githubusercontent.com/372/182180431-08cb04fe-d23a-49e4-a2e5-74fc8e992b34.png)

Reply:
Same with gunicorn, I added `TEST` in the logs:
![image](https://user-images.githubusercontent.com/5695572/182184857-3bcba50a-9838-421d-8605-977eea0c0855.png)

Reply:
I deactivated my venv (per your recommendation in Slack), and it's working now. Thanks!